.github/workflows/test.yml

```diff
@@ -149,24 +151,24 @@ jobs:
           tar -xf aio_objdet_dataset.tar.gz > /dev/null
           
           wget $S3_URL_RESNET_50_V15_TF_FP32 > /dev/null 2>&1
-          IGNORE_DATASET_LIMITS=1 python3 computer_vision/classification/resnet_50_v15/run.py -m resnet_50_v15_tf_fp32.pb -p fp32 -f tf --timeout=60
+          OMP_NUM_THREADS=32 AIO_NUM_THREADS=32 IGNORE_DATASET_LIMITS=1 python3 computer_vision/classification/resnet_50_v15/run.py -m resnet_50_v15_tf_fp32.pb -p fp32 -f tf --timeout=60
```

Comment:
Use envs starting in line 115 for this. No need to specify for each process separately.